index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare module 'svelte-routing' {
+  import { SvelteComponent, SvelteComponentTyped } from 'svelte'
+  
+  interface RouterProps {
+    basepath?: string
+    url?: string
+  }
+
+  interface RouteProps {
```

Comment:
It is possible to pass arbitrary props to `Route`. See readme:

> All properties other than path and component given to the Route will be passed to the rendered component.

So we might need this as well:

```ts
interface RouteProps {
  // ...
  /**
   * Any additional props will be passed to the component passed to the Route
   * via the `component` prop.
   */
  [additionalProp: string]: unknown;
}
```